### DISCUSSION TITLE
JSX Errors in Vue

### DISCUSSION THREAD

**Discussion by jdfwarrior** [storybookjs/storybook#17091] [Help]
I just added Storybook to an existing Vue 3 project yesterday and have noticed that, since doing so, the template tags in all of my SFCs are showing JSX errors when using the `class` attribute on an element, asking if I meant to use `className`. That would be fine... if I were using JSX 😃  That SEEMS to be the issue, that it's incorrectly identifying the code as jsx.

Being a Vue 3 project, I'm using Vite as my bundler. When adding Storybook, I used the instructions found here: [https://storybook.js.org/blog/storybook-for-vite/](https://storybook.js.org/blog/storybook-for-vite/) where I init sb@next and set the builder to storybook-builder-vite.

Is anyone able to point me in a direction as to where I could look to fix this? Reverting all file changes doesn't fix it. I had to remove all the stuff associated with Storybook from node_modules and restart VSCode to resolve the errors.

**Comment by 314171441** [comment]
+1

**Comment by dstuessy** [comment]
I'm having the exact same error since upgrading storybook 6.5 to 7.2.1 this morning. Is there a resolution for this?
